astroquery/simbad/core.py

```diff
@@ -102,7 +112,7 @@ class SimbadClass(BaseVOQuery):
     """
     SIMBAD_URL = 'https://' + conf.server + '/simbad/sim-script'
 
-    def __init__(self, ROW_LIMIT=None):
+    def __init__(self, ROW_LIMIT=None, timeout=None):
```

Comment:
out of scope, but how would you feel for making these explicit?
```suggestion
    def __init__(self, *, ROW_LIMIT=None, timeout=None):
```

Reply:
Maybe I could do `def __init__(self, ROW_LIMIT=None, *, timeout=None):` to avoid a breaking change?

Reply:
sure, that's good, too.